examples/make-db-migration.rs

```diff
@@ -450,25 +508,28 @@ pub fn wasm_migration(updates: &mut Vec<migrations::DbUpdateType>) {
     // wasm_updates[x].0) The WASM hash that is being replaced
     // wasm_updates[x].1) The name of WASM being updated
     // wasm_updates[x].2) The bytes of the new WASM code
-    let wasm_updates: Vec<(&str, &str, &[u8])> =
-        vec![
+    const WASM_UPDATES: [(WasmHash, &str, WasmBytes); 2] = [
         (
             "83afcbf97c35188991ae2e73db2f48cb8d019c4295fe5323d9c3dfebcd5dbec0",
             "tx_transfer.wasm",
-            //include_bytes!("tx_transfer.5c7e44e61c00df351fa7c497cd2e186d71909f1a18db0c8d362dff36057e0fbf.wasm"),
+            // include_bytes!("tx_transfer.
```

Comment:
Can we remove these commented lines?

Reply:
While any byte sequence can be used as the third element of this tuple, I suspect the most common expression here will be of the form `include_bytes("... .wasm")`. Maybe it might be better to replace/expand this line with a remark indicating this expected usage to ensure that users of this example have no doubts about whether they are applying it correctly...

Reply:
Just rewrote the comments, thanks.